### ISSUE TITLE
Get access to the U.S. DB and mimic child records in our DB

### ISSUE THREAD

**Issue by joshuarichard** [joshuarichard/amgg#7]
We need to mimic the records in the U.S. DB of unsponsored children in our test environment after #6 is complete.

**Comment by joshuarichard** [comment]
I spoke with Brian and he said Alex can provide us with access to this DB. I've emailed alex and we're waiting for a response.

**Comment by joshuarichard** [comment]
Got access to this a while ago. going to create a new issue for only adding the child records.
